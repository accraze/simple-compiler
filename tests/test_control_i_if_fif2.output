3
2
9
14
1
2
9
14
3
Error: assign_stmt->op1 is null.
