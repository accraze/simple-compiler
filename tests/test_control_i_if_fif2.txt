
a , b , c, d ;
{
c = 4;
d = 5;
c = c + d;
d = c + d;
IF d > c {
	print a;
	print b;
	print c;
	print d;
}
a = 1  ;
b = 2 ;
IF b > a {
	print a;
	print b;
	print c;
	print d;
}
	
 
IF  b > a {
        a = a+b;
	print a;
	IF a > b { 
		print a;
		b = a+b;
		IF b > a {
			print a;
			print b;
			IF a > b {
				print a;
				print b;
			}
		}
	}
	a = a+b;
	print a;
}
a = a*b;
print a;
print b;
IF a > b {
	print a;
	b = 1;
	a = a+b;
	print a;
	print b;
}

}
