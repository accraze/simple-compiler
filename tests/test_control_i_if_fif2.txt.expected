0
0
9
14
1
2
9
14
3
3
3
5
8
40
5
40
41
1
